apps/ledger-live-mobile/src/logic/internetReachable.ts

```diff
@@ -0,0 +1,14 @@
+import NetInfo from "@react-native-community/netinfo";
+
+export const internetReachable = async () => {
+  // workaround : on iOS, netInfo returns null on the first call, see ; https://github.com/react-native-netinfo/react-native-netinfo/issues/572
+  let isInternetReachable = false;
+
+  isInternetReachable = !!(await NetInfo.fetch()).isInternetReachable;
+
+  if (isInternetReachable !== null) {
```

Comment:
That condition is always true, `isInternetReachable` will never be null in that case because you force it to be a boolean at the previous line.